components/NewsAndUpdatesModal.js

```diff
@@ -0,0 +1,185 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Query } from '@apollo/client/react/components';
+import { useRouter } from 'next/router';
+import { FormattedDate, FormattedMessage } from 'react-intl';
+import styled from 'styled-components';
+import { fontSize, margin } from 'styled-system';
+
+import { API_V2_CONTEXT, gqlV2 } from '../lib/graphql/helpers';
+
+import NextIllustration from './home/HomeNextIllustration';
+import Container from './Container';
+import { Flex } from './Grid';
+import HTMLContent from './HTMLContent';
+import StyledButton from './StyledButton';
+import StyledCarousel from './StyledCarousel';
+import StyledLink from './StyledLink';
+import Modal, { ModalBody, ModalFooter, ModalHeader } from './StyledModal';
+import { Span } from './Text';
+
+const Text = styled.p`
```

Comment:
:ice_cream: I don't get why we need a new component here, most of this can be achieved very easily with the standard `P`, especially since we override the `fontSize` below

Reply:
Ah yes. I just added that as I thought it will be duplicated several times. But I'll change that to a regular `P` tag. 😄